packages/veritone-react-common/src/components/FormBuilder/FormItems/Select.js

```diff
@@ -21,6 +21,8 @@ export default function Select({
     [onChange, name]
   );
 
+  const styles = useStyles({});
```

Comment:
Remove if not used

Reply:
Actually we are using it, so we can not remove it.